src/core/backend/co_spawn.cpp

```diff
@@ -35,19 +37,35 @@ plugin_manager::plugin_manager()
     status = PyConfig_Read(&config);
     if (PyStatus_Exception(status)) {
         console.err("couldn't read config");
-        goto done;
+        PyConfig_Clear(&config);
     }
 
     config.write_bytecode = 0;
     config.module_search_paths_set = 1;
 
+    PyObject* sys_path = PySys_GetObject("path");
+    ssize_t list_size = PyList_Size(sys_path);
+
+    for (size_t i = 0; i < list_size; i++) {
+        PyObject* item = PyList_GetItem(sys_path, i);
+        ssize_t item_size = PyList_Size(item);
+        wchar_t* str = PyUnicode_AsWideCharString(item, &item_size);
+
+        PyWideStringList_Append(&config.module_search_paths, str);
+    }
+    Py_XDECREF(sys_path);

```

Comment:
this snippet requires python to be initialized, before its actually initialized. This is most likely why you needed Py_Initialize() beforehand (which is also invalid)